.github/workflows/validate-nginx-config.yml

```
@@ -0,0 +1,63 @@
+name: Validate Nginx Configuration
+on:
+  workflow_call:
+    inputs:
+      port:
+        default: 8080
+        description: 'Nginx container port number.'
+        required: true
+        type: number
+      realipheader:
+        default: 'X-Forwarded-For' # Ref: https://cloud.google.com/load-balancing/docs/https#target-proxies
```

actionlint (1.7.4)

input "realipheader" of workflow_call event has the default value "X-Forwarded-For", but it is also required. if an input is marked as required, its default value will never be used

(events)